packages/astro/src/template/4xx.ts

```diff
@@ -129,6 +130,21 @@ export function subpathNotUsedTemplate(base: string, pathname: string) {
 	});
 }
 
+export function trailingSlashMismatchTemplate(pathname: string, trailingSlash: 'always' | 'never' | 'ignore') {
+	const corrected =
+		trailingSlash === 'always'
+			? appendForwardSlash(pathname)
+			: removeTrailingForwardSlash(pathname);
+	return template({
+		pathname,
+		statusCode: 404,
+		title: 'Not found',
+		tabTitle: '404: Not Found',
+		body: `<p>Your site is configured with <code>trailingSlash</code> set to <code>${trailingSlash}</code>. Do you want to go to <a href="${corrected}">${corrected}</a> instead?</p>
+<p>Come to our <a href="https://astro.build/chat">Discord</a> if you need help.</p>`,
```

Comment:
Not opposed to also a link to Discord for help, but perhaps a docs link to config reference for `trailingSlash` is more directly helpful?

Reply:
Updated. The message for `base` (which is what I copied this from) should probably also be updated, but that can go ina  different PR